pkg/driver/camera/camera_linux.go

```diff
@@ -192,6 +206,16 @@ func (c *camera) VideoRecord(p prop.Media) (video.Reader, error) {
 
 	cam := c.cam
 
+	// wait up to 5 seconds to read the first frame
+	if err := cam.WaitForFrame(5); err != nil {
+		if _, ok := err.(*webcam.Timeout); ok {
+			return nil, errReadTimeout
+		}
+		// Camera has been stopped.
+		return nil, err
+	}
```

Comment:
What is this aiming for?

Originally, user can read a new frame even after `video.Reader` returned an error, but this change will prevent to try reading after error.

Reply:
The goal was to allow for a longer timeout read error at the beginning. I observed the device I'm using can take a few seconds to read the first frame. I will take this part out.

Reply:
Removed in a8e90965b131100c9202c147c6633bdb7e35bb85